GALLERY.md

```diff
@@ -6,6 +6,7 @@ Here's the list of all the community Blueprints submitted to this repository. Se
 | **Feed Reader with the Friends Plugin**                                | By using the Friends plugin and a CORS proxy, you can read feeds from the web in Playground, and even via ActivityPub                                                                                            | [@akirk](https://github.com/akirk)                         | • [Open in Playground](https://playground.wordpress.net/?blueprint-url=https://raw.githubusercontent.com/wordpress/blueprints/trunk/blueprints/friends-cors/blueprint.json)<br>• [View source](https://github.com/wordpress/blueprints/blob/trunk/blueprints/friends-cors/blueprint.json)<br>• [Edit](https://playground.wordpress.net/builder/builder.html?blueprint-url=https://raw.githubusercontent.com/wordpress/blueprints/trunk/blueprints/friends-cors/blueprint.json)                                                                                        |
 | **Stylish Press**                                                      | A Woo store with custom theme, content, and products.                                                                                                                                                            | [@adamziel](https://github.com/adamziel)                   | • [Open in Playground](https://playground.wordpress.net/?blueprint-url=https://raw.githubusercontent.com/wordpress/blueprints/trunk/blueprints/stylish-press/blueprint.json)<br>• [View source](https://github.com/wordpress/blueprints/blob/trunk/blueprints/stylish-press/blueprint.json)<br>• [Edit](https://playground.wordpress.net/builder/builder.html?blueprint-url=https://raw.githubusercontent.com/wordpress/blueprints/trunk/blueprints/stylish-press/blueprint.json)                                                                                     |
+| Demo of Twenty-Twenty-Five                                                    | Blueprint with demo content for the Twenty-Twenty-Five                                                                                                                                              | [@bph](https://github.com/bph) | • [Open in Playground](https://playground.wordpress.net/?blueprint-url=https://raw.githubusercontent.com/wordpress/blueprints/trunk/blueprints/tt5-demo/blueprint.json)<br>• [View source](https://github.com/wordpress/blueprints/blob/trunk/blueprints/tt5-demo/blueprint.json)<br>• [Edit](https://playground.wordpress.net/builder/builder.html?blueprint-url=https://raw.githubusercontent.com/wordpress/blueprints/trunk/blueprints/tt5-demo/blueprint.json)
```

Comment:
Should the word "Theme" follow "Blueprint with demo content for the Twenty-Twenty-Five"?

Reply:
Yes, somehow my copy/pasta got truncated.. fixed now.